tests/integration/questionnaire/test_questionnaire_household_composition.py

```diff
@@ -0,0 +1,29 @@
+from tests.integration.create_token import create_token
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestQuestionnaireAddAnswer(IntegrationTestCase):
+
+    BASE_URL = '/questionnaire/test/household_question/789/'
+
+    def setUp(self):
+        super().setUp()
+        token = create_token('household_question', 'test')
+        self.client.get('/session?token=' + token.decode(), follow_redirects=False)
+        post_data = {
+            'action[start_questionnaire]': 'Start survey'
+        }
+        self.household_composition_url, resp = self.postRedirectGet(self.BASE_URL + 'introduction', post_data)
+
+    def test_add_person_skips_validation(self):
+        # Given
+        post_data = {
+            "action[add_answer]": ""
+        }
+
+        # When
+        resp = self.client.post(self.household_composition_url, data=post_data)
+
+        # Then
+        self.assertEqual(200, resp.status_code)
+        self.assertNotIn('This field is mandatory.', resp.get_data(True))
```

Comment:
If we returned a status code of 400 for invalid answers we wouldn't need to check the body making testing end points nicer. I know in REST apis status codes are ubiquitous but not sure how it fits in outside them